### ISSUE TITLE
add user block function

### ISSUE THREAD

**Issue by Jamelle-Boose** [Jamelle-Boose/twitter-streamer#19]

**Comment by Jamelle-Boose** [comment]
`wontfix`

Users can already be blocked by adding a rule. However, keywords must be included since rules consisting of only negating operators are not permitted.

[Building rules for filtered stream - Boolean operators and grouping](https://developer.twitter.com/en/docs/twitter-api/tweets/filtered-stream/integrate/build-a-rule#build)

![image](https://user-images.githubusercontent.com/19831942/182087867-2ea1891b-24b3-447a-8963-4bf5bc9ff9b6.png)
